src/update.c

```diff
@@ -278,20 +339,57 @@ int main_update()
 		goto clean_curl;
 	}
 
+	mix_exists = check_mix_exists();
+
 	fprintf(stderr, "Update started.\n");
 
 	grabtime_start(&times, "Update Step 1: get versions");
 
-	read_subscriptions_alt(&current_subs);
-
-	/* Step 1: get versions */
+	read_subscriptions(&current_subs);
 
+/* Step 1: get versions */
+version_check:
 	ret = check_versions(&current_version, &server_version, path_prefix);
 
 	if (ret < 0) {
 		ret = EXIT_FAILURE;
 		goto clean_curl;
 	}
+
+	if (mix_exists) {
+		ret = check_mix_versions(&mix_current_version, &mix_server_version, path_prefix);
+		if (ret < 0) {
+			ret = EXIT_FAILURE;
+			goto clean_curl;
+		}
+		/* Check if a new upstream version is available so we can update to it still */
+		if (need_new_upstream(server_version)) {
+			printf("NEW CLEAR AVAILABLE %d\n", server_version);
+			ret = check_manifests_uniqueness(server_version, mix_server_version);
+			if (ret) {
+				printf("\n\t!! %i collisions were found between mix and upstream, please re-create mix !!\n", ret);
+			}
```

Comment:
So, this seems like an error, but you don't actually error here. This should also go to stderr.

Reply:
It's not so much an error as it is us saying what we explicitly support right now. I can make it overridable with a config, but I want people to be able to still add conflicting content and just be told if that is the case.